### PR TITLE
Remove PerformanceStats/Sample

### PR DESCRIPTION
**Pull request by notspiff**
Minimal impact, but still I think it's cleaner.

### CODE REVIEW

**Comment by FernetMenta** [comment]
IMO those performance measures are pretty useless and I would drop them completely. If it is desired to keep them, please drop the ifdefs. There is zero benefit in not compiling/linking those files.

**Comment by notspiff** [comment]
i agree this is useless code, it is an xbox artifact where this class reported back to vs over a dedicated bus. so i am fine with dropping. any objections ?

File: xbmc/utils/PerformanceSample.cpp [hunk 1]
```diff
@@ -28,7 +28,7 @@
 #endif
 
 #ifdef HAS_PERFORMANCE_SAMPLE
-#include "Application.h"
```
Comment (FernetMenta) [inline xbmc/utils/PerformanceSample.cpp]: a define HAS_PERFORMANCE_SAMPLE in PerformanceSample.cpp makes not much sense in the times of cmake.

**Comment by notspiff** [comment]
i will prepare scrapping of this code and update pr if nobody objects soon.

**Comment by notspiff** [comment]
jenkins build this please

And succeed this time dammit

**Comment by notspiff** [comment]
jenkins build this please

**Comment by notspiff** [comment]
jenkins build this please

**Comment by notspiff** [comment]
no protests. this goes in now.

**Pull request merged**